### PR TITLE
Rf7ECYx3-CRUD operation for route

### PR DESCRIPTION
**Pull request by Julesbi**
# Crud operation for routes

## Overview
In this feature, I have completed CRUD operation for routes where an operator can be able

 - to create
 - update
  - delete a route, and
  - users are able to view the route's details 
## Testing 

To test this follow these steps 

1. Pull this branch
2. run `npm install` to install all dependencies in `package.json`
3. run `touch .env` to create .env file on the root folder
4. Based on database configuration enter your DB_URL
5. run `npm run migrate` to create the database
6. run `npm build` to build the app with babel
7.  run `npm run dev` to start server 
8. use postman to test API

## Relevant Pivotal tracker
[CRUD operation for routes](https://trello.com/c/Rf7ECYx3/7-crud-operations-for-routes)

### CODE REVIEW

**Comment by NyiingangoPacifique** [comment]
i can't see review i don't know if its me only

**Comment by Julesbi** [comment]
@NyiingangoPacifique  I have added you.

File: src/controllers/user.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+              .then(() => res.status(200).send({
+                message: 'user successfully deleted'
+              }))
+              .catch(error => res.status(400).send(error));
+          })
+          .catch(error => res.status(400).send(error))
+      }
+}
+
+//export default Users; */
```
Comment (christianR15c) [inline src/controllers/user.js]: why is this file commented?

File: test/route.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                /* response.body.should.have.property('id').eq(1); */
+
+            done();
+            })
+        })
+    })
+
+    /* Test the POST route */
+
+
```
Comment (christianR15c) [inline test/route.js:77]: Why are these comments are here and empty? if they reference or explaining something should be indicated

**Review by christianR15c — CHANGES_REQUESTED** [review]

File: test/route.js (hunk 2 above)
Comment (Julesbi) [reply test/route.js:77]: I'm still working on some tests on routes.

File: src/controllers/user.js (hunk 1 above)
Comment (Julesbi) [reply src/controllers/user.js]: Was a mistake, thanks  let me correct it

File: config/config.js [hunk 3]
```diff
@@ -1,36 +1,17 @@
-require('dotenv').config();
+require('dotenv').config()
```
Comment (NyiingangoPacifique) [inline config/config.js:1]: work on this duplicate of dotenv config

File: config/config.js [hunk 4]
```diff
@@ -1,36 +1,17 @@
...
 
-const env = process.env.NODE_ENV || 'dev';
-
-const dev = {
-  app: {
-    port: parseInt(process.env.APP_PORT),
-  },
-  db: {
-    host: process.env.HOST,
```
Comment (NyiingangoPacifique) [inline config/config.js:10]: better use the config we agred as team last night

**Pull request closed without merging**